user_guide/data_management.rst

```diff
@@ -0,0 +1,234 @@
+.. _user-guide-data-management-basics:
+
+Data Management Basics
+======================
+
+With OroCRM, you can record, store, process and analyze various customer-related information. This information may cover 
+different areas including personal and contact details, records of the customer activity, possible sales and 
+arrangements, and many others. This can be any type of information necessary to monitor, manage and understand specific 
+customer-related activities. 
+
+In this article we will overview the ways data is represented and managed in OroCRM. For 
+each point that requires further explanation, you will find links to related documentation. 
+
+
+How the data is organized
+-------------------------
+
+The customer relationship management process requires us to collect and process huge amount of information, and OroCRM 
+is sharpened to do it in the most efficient and convenient way.
+
+Technically, a collection of information of a similar nature is represented in OroCRM with an **Entity**. Each instance 
+of this collection is called a **Record**. Details of each record are its **Properties**.
+
+For example, information about customers of a Web-store is collected as properties of the records of the Web Customer 
+entity. This means, that one of the OroCRM entities is “Web Customer” and the system knows that records of this 
+entity can have a number of properties, such as the first name, last name, email, phone number, id of the shipping 
+address, id of the cart and ids of the orders. Moreover, the system is aware that some of this properties must be 
+defined and some are optional. When a new Web customer is added to the system – its record is created and values of 
+the properties are save for it. If something has changed (e.g. the phone number), the properties are updated. 
```

Comment:
Edit: Information about customers of an online store is collected as properties of the records of the Web Customer entity. This means that one of the OroCRM entities is “Web Customer” and the system knows that records of this entity can have a number of properties, such as the first name, last name, email, phone number, ID of the shipping address, ID of the cart, and ID's of the orders. The system is also aware that some of these properties need to be defined while some are optional. When a new customer is added to the system, their record is created and values of the properties are saved for it. If something has changed (e.g., the phone number), the properties are updated.